buildspecs/linux_riscv64.spec

```diff
@@ -0,0 +1,260 @@
+		<flag id="env_littleEndian" value="true"/>
+		<flag id="gc_batchClearTLH" value="true"/>
+		<flag id="gc_debugAsserts" value="true"/>
+		<flag id="gc_idleHeapManager" value="true"/>
+		<flag id="gc_inlinedAllocFields" value="true"/>
+		<flag id="gc_minimumObjectSize" value="true"/>
+		<flag id="gc_tlhPrefetchFTA" value="true"/>
+		<flag id="graph_cmdLineTester" value="true"/>
+		<flag id="graph_compile" value="true"/>
+		<flag id="graph_enableModularityTesting" value="false"/>
+		<flag id="graph_enableTesting" value="false"/>
+		<flag id="graph_enableTesting_Java8" value="true"/>
+		<flag id="graph_includeThrstatetest" value="true"/>
+		<flag id="graph_j2seSanity" value="true"/>
+		<flag id="graph_jgrinder" value="true"/>
+		<flag id="graph_plumhall" value="true"/>
+		<flag id="graph_useJTCTestingPlaylist" value="true"/>
+		<flag id="graph_verification" value="true"/>
+		<flag id="interp_aotCompileSupport" value="true"/>
+		<flag id="interp_aotRuntimeSupport" value="true"/>
+		<flag id="interp_debugSupport" value="true"/>
+		<flag id="interp_enableJitOnDesktop" value="true"/>
+		<flag id="interp_flagsInClassSlot" value="true"/>
+		<flag id="interp_gpHandler" value="true"/>
+		<flag id="interp_growableStacks" value="true"/>
+		<flag id="interp_hotCodeReplacement" value="true"/>
+		<flag id="interp_nativeSupport" value="true"/>
+		<flag id="interp_profilingBytecodes" value="true"/>
+		<flag id="interp_sigQuitThread" value="true"/>
+		<flag id="interp_useUnsafeHelper" value="true"/>
+		<flag id="ive_jxeFileRelocator" value="true"/>
+		<flag id="ive_jxeInPlaceRelocator" value="true"/>
+		<flag id="ive_jxeNatives" value="true"/>
+		<flag id="ive_jxeOERelocator" value="true"/>
+		<flag id="ive_jxeStreamingRelocator" value="true"/>
+		<flag id="ive_romImageHelpers" value="true"/>
+		<flag id="jit_classUnloadRwmonitor" value="true"/>
+		<flag id="jit_dynamicLoopTransfer" value="true"/>
+		<flag id="jit_fullSpeedDebug" value="true"/>
+		<flag id="jit_gcOnResolveSupport" value="true"/>
+		<flag id="jit_newDualHelpers" value="true"/>
+		<flag id="jit_newInstancePrototype" value="true"/>
+		<flag id="jit_supportsDirectJNI" value="true"/>
+		<flag id="module_algorithm_test" value="true"/>
+		<flag id="module_bcutil" value="true"/>
+		<flag id="module_bcverify" value="true"/>
+		<flag id="module_cassume" value="true"/>
+		<flag id="module_cfdumper" value="true"/>
+		<flag id="module_codegen_common" value="false"/>
+		<flag id="module_codegen_riscv" value="false"/>
+		<flag id="module_codegen_ilgen" value="false"/>
+		<flag id="module_codegen_opt" value="false"/>
+		<flag id="module_codert_common" value="false"/>
+		<flag id="module_codert_riscv" value="false"/>
+		<flag id="module_codert_vm" value="false"/>
+		<flag id="module_ddr" value="true"/>
+		<flag id="module_ddr_gdb_plugin" value="true"/>
+		<flag id="module_ddrext" value="true"/>
+		<flag id="module_gdb" value="true"/>
+		<flag id="module_gdb_plugin" value="true"/>
```

Comment:
Why are these flags set (they're not set for linux_x86-64)?

Reply:
Removed as there is no difference on RISC-V as compared to other platforms.